views/blocks/careers.twig

```diff
@@ -0,0 +1,3 @@
+<ul class='list--arrow'>
```

Comment:
Move your test for emptiness here and placeholder text here. it will be something like:
```twig
{% if content|trim %}
    <ul class='list--arrow'>
      {{content}}
    </ul>
{% else %}
    <p>{{attributes.placeholder? attributes.placeholder: "there are no jobbies right now" }}</p>
{% endif %}
```